client/components/domains/register-domain-step/index.jsx

```diff
@@ -71,77 +72,40 @@ function getQueryObject( props ) {
 	};
 }
 
-function processSearchStatQueue() {
```

Comment:
By removing this whole function, you've undone Umur's work from https://github.com/Automattic/wp-calypso/pull/4941. The idea was prevent sending the stats to the backend while the user is slowly typing. So if I type `slowlytyping`, but make a pause here and there (because I'm on mobile or just a slow typer), we shouldn't report partial queries like `slowl` or `slowlytyp`, but the longest one from a 10s window. See the PR for more details. Not sure if this was a deliberate decision or not, but I think it reduces noise in our stats.